src/legacy/core_plugins/tile_map/public/coordinate_maps_visualization.js

```diff
@@ -214,6 +214,24 @@ export function CoordinateMapsVisualizationProvider(Notifier, Private) {
             '1': geoBoundsAgg.toDsl()
           };
         });
+
+        // This is a temporary solution to ensure that global queries & filters
+        // are included in searchSource when querying the geohash bounds.
+        // TODO: Remove this as a part of elastic/kibana#30593
+        const { filters, query } = this.vis.API.__UNSTABLE_GLOBAL_STATE_DATA__;
+        if (query) {
+          searchSource.setField('query', query);
+        }
+        searchSource.setField('filter', () => {
+          const activeFilters = [...filters];
+          const indexPattern = agg.getIndexPattern();
+          const useTimeFilter = !!indexPattern.timeFieldName;
+          if (useTimeFilter) {
+            activeFilters.push(this.vis.API.timeFilter.createFilter(indexPattern));
+          }
```

Comment:
This solution seemed to do the trick for time filters, without the `timeRange` needing to be passed down as well, but let me know if you can find any holes in this approach. It is mostly stolen from `input_control_vis`.